06-release-engineering-analytics.Rmd

```diff
@@ -73,8 +74,8 @@ These initial papers were @adams2016a, @da2016a, @d2014a, @khomh2012a, and
 @khomh2015a.
 
 We collected publications using two search engines: Scopus and Google Scholar.
-These each encompass various databases such as ACM Digital Library, Springer,
-IEEE Xplore and ScienceDirect.
+Each of the two search engines encompass several databases such as
```

Comment:
I agree, this is better. Ayushi also suggested replacing the verb "encompass" with "comprise maybe that would be even better?